features/admin.applications.v1/components/forms/advanced-configurations-form.tsx

```diff
@@ -325,7 +325,7 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
                                 </Grid.Column>
                                 <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
                                     <Heading as="h4">
-                                        { "Application native authentication" }
+                                        { "App-Native Authentication" }
```

Comment:
Hope the casing of `App-Native` is verified.

Reply:
Verified with the doc team.